src/app/components/Transaction/index.tsx

```diff
@@ -479,12 +481,33 @@ export function Transaction(props: TransactionProps) {
           <Text weight="bold" size={isMobile ? 'medium' : 'xlarge'}>
             <AmountFormatter amount={transaction.amount!} smallTicker />
           </Text>
-          <Text color={transaction.status ? 'successful-label' : 'status-error'} size="small" weight="bold">
-            {transaction.status ? (
-              <span>{t('account.transaction.successful', 'Successful')}</span>
-            ) : (
-              <span>{t('account.transaction.failed', 'Failed')}</span>
-            )}
+          <Text
+            color={(() => {
+              switch (transaction.status) {
+                case true:
+                  return 'successful-label'
+                case false:
+                  return 'status-error'
+                case undefined:
+                default:
+                  return 'status-warning'
+              }
+            })()}
+            size="small"
+            weight="bold"
+          >
+            {(() => {
+              switch (transaction.status) {
+                case true:
+                  return <span>{t('account.transaction.successful', 'Successful')}</span>
+                case false:
+                  return <span>{t('account.transaction.failed', 'Failed')}</span>
+                case undefined:
+                default:
+                  /* TODO: Add translation */
+                  return <span>Pending</span>
```

Comment:
This could add confusion
https://github.com/oasisprotocol/explorer/blob/3d82e749424e6d9001041948a8892c126d54dcab/src/oasis-nexus/generated/api.ts#L918-L921
undefined success in nexus api means unknown / confidential

Reply:
Refactored status from `boolean` to `enum` here https://github.com/oasisprotocol/oasis-wallet-web/pull/1970